packages/@aws-cdk/aws-servicecatalog/lib/tag-options.ts

```diff
@@ -1,14 +1,40 @@
+import * as cdk from '@aws-cdk/core';
+import { hashValues } from './private/util';
+import { InputValidator } from './private/validation';
+import { CfnTagOption } from './servicecatalog.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from 'constructs';
+
 /**
- * Defines a Tag Option, which are similar to tags
- * but have multiple values per key.
+ * Defines a set of TagOptions, which are a list of key-value pairs managed in AWS Service Catalog.
+ * It is not an AWS tag, but serves as a template for creating an AWS tag based on the TagOption.
  */
 export class TagOptions {
```

Comment:
Okay, I had this before but removed it - the reason being is that there is not really an `id` since there isn't an actual umbrella resource, this just creates a list of ones automatically.  

Or, would this suggest that perhaps passing in the `id` to `TagOptions` I could then pass that along to each of the underlying ones so that it becomes part of its path?  I could see that being useful if you are ever making multiple ones and are looking at the end resources.

Reply:
You don't have to pass the `id` anywhere - if you create the underlying resources under `this`, that identifier will automatically become part of their unique ID (as it will be part of their path in the construct tree).

Reply:
whoops I think I meant to say that the id is implicitly passed along. but yes you are correct.